### PR TITLE
Create a bio for jpalleoo

### LINKED ISSUE #43: Add bio for jpallepo
### Things to inculde
**This is my test comment**
- [] vacation spots

@githubstudent should I add something

### PR DESCRIPTION
**Pull request by jpallepo**
Fixes #43 

Adding a bio fro jpallepo

@tallus what else

### CODE REVIEW

File: bios/jpallepo-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## jpallepo 
+
+- loves beach
+- likes travelling
```
Comment (tallus) [inline bios/jpallepo-bio.md]: Anywhere you have been recently you liked

File: bios/jpallepo-bio.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## jpallepo 
+
+- loves beach
```
Comment (JeffreyRRoss) [inline bios/jpallepo-bio.md]: What's the name of your favorite beach?

**Comment by jpallepo** [comment]
:shipiti

**Comment by tallus** [comment]
:+1:

**Comment by jpallepo** [comment]
:shipit

**Pull request merged**